src/main/java/org/opensearch/security/tools/ChecksumCalculator.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Why do we need a tool for this? There are many command line tools available for this kind of purpose, e.g. https://stackoverflow.com/questions/3358420/generating-a-sha-256-hash-from-the-linux-command-line

Reply:
Thought it'd be of value to add capability to generate shas of all files in a folder with a given extension. Otherwise they'd have to cat each file manually and run sha256sum. i.e. `cd <cert-folder>;  cat cert.pem | sha256sum`

But the main purpose was to seek feedback on PR whether we should ship with this or not. I felt it might be useful down the line so I added it.

Reply:
If instead this tool that generated the certs and then updated all of the relevant locations (including creating the checksums) - I would be onboard.